packages/scripts/src/admin-proposals/transferTokensFromGovernor.js

```diff
@@ -35,11 +34,17 @@ async function run() {
   const web3Providers = { 1: getWeb3ByChainId(1) }; // netID => Web3
   const web3 = web3Providers[1];
   const tokenAddress = await _getContractAddressByName("VotingToken", 1);
+  const governorAddress = await _getContractAddressByName("Governor", 1);
   console.log("tokenAddress:", tokenAddress);
   const uma = new web3.eth.Contract(ExpandedERC20.abi, tokenAddress);
 
   const gasEstimator = await setupGasEstimator();
 
+  var originalBalance = await uma.methods.balanceOf(recipient).call();
+  const originalGovernorBalance = await uma.methods.balanceOf(governorAddress).call();
```

Comment:
nit: could promise.all these to be faster :D

Reply:
also don't use `var`

Reply:
Can you give an example of how this would be re-written to use Promise.all()?

Reply:
Nevermind, added some code using Promise.all(), lemme know what you think!